components/EditDeleteInfoForm/index.js

```diff
@@ -134,7 +133,7 @@ export default function EditDeleteInfoForm({ entry, spotId }) {
                 )}
               </EntryEditButton>
               <EntryDeleteButton
-                onClick={handleDelete}
+                onClick={openModal}
                 aria-label={`Delete entry for ${entry.info}`}
                 disabled={disabled}
```

Comment:
Minor thing, and not requesting any changes but... the exact appearance of disabled buttons can vary slightly between different browsers and versions. If you want to ensure a consistent and specific visual style for disabled buttons across different browsers, you can use CSS to customise the styling